### PR TITLE
add stop_process_execution

### PR DESCRIPTION
**Pull request by zklgame**

### CODE REVIEW

File: api-schema/xdb.yaml [hunk 1]
```diff
@@ -32,7 +32,25 @@ paths:
...
+      requestBody:
+        content:
+          'application/json':
+            schema:
+              $ref: '#/components/schemas/ProcessExecutionStopRequest'
+      responses:
+        '200':
+          description: successful operation
+          content: {}
+        '400':
```
Comment (longquanzheng) [inline api-schema/xdb.yaml:48]: we also need to return 404 when the process is not existing or not running. (this is the same experience in iWF/Cadence/Temporal). Though in iWF, it's using subStatus in errorResponse. I want to improve it to use HTTP status to be more clear

**Review by longquanzheng — COMMENTED** [review]
good work so far!

**Review by longquanzheng — APPROVED** [review]

**Comment by longquanzheng** [comment]
when you are ready, you can merge this one first and I will rebase my PR.

**Comment by zklgame** [comment]
> when you are ready, you can merge this one first and I will rebase my PR.

Merged!

**Pull request merged**